core/Rename.cpp

```diff
@@ -142,10 +258,27 @@ namespace olympia
                     const auto rf  = determineRegisterFile(src);
                     const auto num = src.field_value;
                     auto & bitmask = renaming_inst->getSrcRegisterBitMask(rf);
-                    bitmask.set(num);
+                    uint32_t prf;
```

Comment:
Always initialize your variables to some invalid value -- in case someone modifies your code and forgets to provide a final value.  (most compilers catch that, but not always...)

Reply:
Addressed in [1bf68f4](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/1bf68f4f79753f81792e51ef39c2b06e261e82de)